android/hybridcommon/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -44,6 +44,12 @@ fun StoreProduct.map(): Map<String, Any?> =
         "currencyCode" to priceCurrencyCode,
         "introPrice" to mapIntroPrice(),
         "discounts" to null,
+        "pricePerWeek" to pricePerWeek()?.amountMicros,
+        "pricePerMonth" to pricePerMonth()?.amountMicros,
+        "pricePerYear" to pricePerYear()?.amountMicros,
+        "pricePerWeekString" to pricePerWeek()?.formatted,
+        "pricePerMonthString" to pricePerMonth()?.formatted,
+        "pricePerYearString" to pricePerYear()?.formatted,
```

Comment:
Ohh right these are only in the `StoreProduct` level right now... I wonder if we should add something similar at the `PricingPhase` level as well in Android (which is used inside the `SubscriptionOption` model). These are also useful though, so not needed for this PR.

Reply:
Good idea! As discussed, this would need some changes to purchases-android first, to add the other periods besides monthly, so I'll leave them out of this PR.